src/ui/AdminMessage/__tests__/AdminMessage.spec.js

```diff
@@ -1,25 +1,19 @@
 import React from 'react';
-import { shallow } from 'enzyme';
-import renderer from 'react-test-renderer';
+import { render, screen } from '@testing-library/react';
 
 import AdminMessage from "../index";
 import dummyAdminMessage from '../adminMessageDummyData.mock';
 
 describe('AdminMessage', () => {
-  it('should contain className', function () {
+  it.only('should contain className', function () {
```

Comment:
it.only should skip other tests, no? Any reason?

Reply:
This is my mistake sorry and thanks